api-report/map.api.md

```diff
@@ -65,6 +67,8 @@ export interface IDirectoryDataObject {
 export interface IDirectoryEvents extends IEvent {
     // (undocumented)
     (event: "containedValueChanged", listener: (changed: IValueChanged, local: boolean, target: IEventThisPlaceHolder) => void): any;
+    // (undocumented)
+    (event: "dispose", listener: (target: IEventThisPlaceHolder) => void): any;
```

Comment:
Is it dispose or disposed? Do we have interface that we can inherit to get proper signature, instead of defining it everywhere?

Reply:
Changed to disposed and made the directory implement IDisposable interface.